core/base/workspace_aliases.hpp

```diff
@@ -0,0 +1,255 @@
+// SPDX-FileCopyrightText: 2017 - 2024 The Ginkgo authors
+//
+// SPDX-License-Identifier: BSD-3-Clause
```

Comment:
I really think this should be different. You can add this file to the exludes in the pre-commit config for the format-headers hook. Then the changes to it won't get overwritten anymore.